components/Hero.vue

```diff
@@ -271,6 +272,16 @@ $body-font: "Source Sans Pro", sans-serif;
   }
 }
 
```

Comment:
It is also covering the stars. Is it a big deal (personally idrk)?
![image](https://user-images.githubusercontent.com/40312631/99494273-94afa580-2925-11eb-9608-b06f8d284df0.png)

If you think it's a little bit iffy, you could try shifting the stars up by a bit? Otherwise, it's fine.

Reply:
i think it's ok!